core.js

```diff
@@ -2878,6 +2896,10 @@ class HTMLStyleElement extends HTMLLoadableElement {
     return running;
   }
 }
+const _shouldRequire = url => (url.endsWith('.tgz') ||
```

Comment:
This should probably just check the `Content-Type`.

Reply:
That might be dangerous. A cursory google suggests that .tar.gz files are often misconfigured server-side: https://github.com/haskell/cabal/issues/615

The thing I'm worried about is, .tar.gz isn't actually a "tar-gz", it's "gzipped data". Which happens to be exactly the same type of data that servers usually send down, even if it's just text and not a tarball.

I'm sure there's probably a way to decompress the stream in memory and check whether it's a tarball, but... there's no situation where <script src="foo.tar.gz"></script> means anything other than require it as a tarball.

Reply:
Can we skip this entirely and if the type is application/require we assume it’s a tarball?

Reply:
Yeah. I wanted to, but for some reason setting `type="application/require"` causes the data not to fetch. Not sure why yet. That's when I switched to filename extension detection.

Reply:
Htmlscriptekement follows spec here. This new type would need to be whitelisted for it to load. That seems correct behavior.